shell/platform/windows/keyboard_key_handler.cc

```diff
@@ -129,9 +129,9 @@ void KeyboardKeyHandler::DispatchEvent(const PendingEvent& event) {
 #else
   char32_t character = event.character;
 
-  if (event.action == WM_SYSKEYDOWN || event.action == WM_SYSKEYUP) {
-    return;
-  }
+  // Sys key events can't be synthesized. Their dispatching should be prevented
+  // in earlier code.
+  assert(event.action != WM_SYSKEYDOWN && event.action != WM_SYSKEYUP);
```

Comment:
You could make the message part of the assert message:
```suggestion
  assert(event.action != WM_SYSKEYDOWN && event.action != WM_SYSKEYUP
    && "Sys key events can't be synthesized. Their dispatching should be prevented in earlier code.");
```

Then the message will also print when the assert fires.